geode-junit/src/main/java/org/apache/geode/cache/ssl/CertificateBuilder.java

```diff
@@ -79,16 +77,27 @@ public CertificateBuilder(int days, String algorithm) {
   }
 
   private static GeneralName dnsGeneralName(String name) {
-    return new GeneralName(GeneralName.dNSName, name);
+    try {
+      return new GeneralName(new DNSName(name));
+    } catch (IOException ex) {
+      throw new UncheckedIOException(ex);
+    }
```

Comment:
The value of bundling this as into a runtime exception is that this method can still be used in contexts (e.g. Stream.map(f)`) where a method throwing a checked exception would not be allowed. Ok.